OpenOrchestrator/scheduler/runner.py

```diff
@@ -224,14 +231,15 @@ def run_process(trigger: Trigger) -> subprocess.Popen | None:
         trigger: The trigger whose process to run.
 
     Returns:
-        subprocess.Popen: The Popen instance of the process if successful.
+        Job: A Job object referencing the process, if succesful.
```

Comment:
```suggestion
        Job: A Job object referencing the process if succesful.
```

That comma seems out of place to me.